From d8b0e6f20d3e4bfc1989e98856ab11a16851b32e Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Tue, 15 Sep 2020 20:13:44 -0400
Subject: [PATCH 2/2] runtest1.sh: unset PAGER while running tests.

There's no need to page this output, and it makes automating the test
suite a pain in the butt. Unsetting PAGER in the test script makes
it work normally again.

Gentoo-bug: https://bugs.gentoo.org/742443
---
 runtest1.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/runtest1.sh b/runtest1.sh
index 2dfadd1..8d5c46d 100755
--- a/runtest1.sh
+++ b/runtest1.sh
@@ -1,4 +1,5 @@
 #!/bin/sh
 
 A="$(command -v gawk || command -v awk)"
+unset PAGER
 $A "BEGIN{for (i=0;i<10;++i) print rand()*10}" | ./xjobs -j8 ./testdummy
-- 
2.26.2

